mailer/mailer.go

```diff
@@ -13,7 +13,7 @@ import (
 // Mailer defines the interface a mailer must implement.
 type Mailer interface {
 	Send(user *models.User, subject, body string, data map[string]interface{}) error
-	InviteMail(user *models.User, referrerURL string) error
+	InviteMail(user *models.User, referrerURL string, notSendMail bool) (string, error)
```

Comment:
@awalias you happy with the param name here? Could be a bit confusing `notSendMail`, but I don't know of a better approach given the structure of the invite/mailer system

Reply:
I know that it's ugly. On the other hand I can implement return invite url without breaking interface, but we will have duplicate logic for generating invite link and I think that it's more painfull then ugly interface.

Reply:
@awalias answer please

Reply:
Hey @icecream78 - we're just wrapping up Launch Week so the team has been a bit busy. We'll get to this soon .